hubot-rocketchat/README.md

```diff
@@ -61,6 +61,11 @@ Read the [Getting Started](https://github.com/tolstoyevsky/mmb#getting-started)
   <tr>
     <td align="center" colspan="3"><b>Rocket.Chat Hubot adapter</b></td>
   </tr>
+  <tr>
+    <td>AUTH_ATTEMPTS</td>
+    <td>Sets attempts number for bot authentication trying before script running</td>
```

Comment:
Specifies the number of authentication attempts before running the bot. The parameters helps the container to be put on pause when, for example, the bot account is being prepared.